### DISCUSSION TITLE
Does it support Outlook?

### DISCUSSION THREAD

**Discussion by hanifmhd** [wmcmahan/react-native-calendar-events#371] [Q&A]
Hi guys,

I wanna ask, does it support Outlook calendar? I don't see the example of outlook in readme file.

Thank you.

**Comment by MoOx** [comment]
This library only support built-in OS calendar support.
It is not tied to a particular app, but to the OS native calendars list that you can have on iOS or Android.

Exchange calendars (which are the preferred thing in the microsoft/outlook world) are supported since iOS and Android support them.

I hope that's clear for you.
